create_toy_data.py

```diff
@@ -62,42 +65,115 @@ def load_data(
                     obj that returns the content
                     for each document.
 
-        num_docs : inn / None
+        num_docs : int / None
                    number of documents to process
                    if None, return all
 
         return_tags : boolean
                       if true, tags are returned w/
                       words in a tuple
 
+        return_sent_labels : boolean
+                            if true, a boolean is
+                            returned for whether the
+                            token repr. EOS.
         Returns
         -------
         data : Python generator
                yields a single token (w/ POS)
     '''
 
+    default_splitter = nltk.data.load(
+        'tokenizers/punkt/english.pickle')
+
```

Comment:
So by switching to the punkt tokenizer, we're getting more "bad splits":

reading: datasets/brown/raw/cl11
found an item with no tag: M.E.

reading: datasets/brown/raw/cj24
found an item with no tag: B.
found an item with no tag: B.

reading: datasets/brown/raw/cj58
found an item with no tag: B.

reading: datasets/brown/raw/cj73
found an item with no tag: '?

reading: datasets/brown/raw/cf29
found an item with no tag: TR.

reading: datasets/brown/raw/ca15
found an item with no tag: U.

reading: datasets/brown/raw/cd03
found an item with no tag: B.B.C.

reading: datasets/brown/raw/ca40
found an item with no tag: A.A.U.